src/server/i18n/bg.json

```diff
@@ -0,0 +1,241 @@
+{
+  "site_title": "DVSA roadside fines portal",
+  "feedback_banner": "Това е нова услуга — Вашата обратна връзка ще ни помогне да я подобрим.",
+  "breadcrumbs": {
+    "home": "Начална страница",
+    "pay_dvsa_fine": "Платете глоба към Агенцията за стандарти на водачи и автомобили (DVSA)",
+    "payment_code": "Код за плащане",
+    "fines_summary": "Fines summary",
+    "court_deposit": "Съдебен депозит",
+    "immobilisation_fee": "Глоба за спиране от движение",
+    "fixed_penalty": "Фиксирана глоба",
+    "receipt": "Receipt",
+    "penalty_details": "Данни за глобата",
+    "payment": "Плащане",
+    "active_session": "Active session warning",
+    "cookie_preferences": "Cookie preferences",
+    "cookie_details": "Cookie details"
+  },
+  "cookie_banner": {
+    "title": "Tell us whether you accept cookies",
+    "message": "We use <a href='/cookie-details'>cookies to collect information</a> about how you use the Roadside Payments service. We use this information to make the website work as well as possible and improve government services.",
+    "button_accept": "Accept all cookies",
+    "button_preferences": "Set cookie preferences"
+  },
+  "languages": {
```

Comment:
What is this list used for? Should it contain the new languages?

Reply:
It should do but this is only appears to be used on the start page which is not used. Also, we don't have the translations for each new language in the language of each new language anyway (if that makes sense)